scripts/devnet/docker-compose.yml

```
@@ -181,6 +181,7 @@ services:
     environment:
       - FIL_PROOFS_PARAMETER_CACHE=${FIL_PROOFS_PARAMETER_CACHE}
       - RUST_LOG=info,forest_filecoin::blocks::header=trace
+      - FOREST_F3_PERMENANT_PARTICIPATING_MINER_ADDRESSES=${MINER_ACTOR_ADDRESS}
```

typo, I saw it also somewhere else at some point; please ensure all of those are corrected.

```suggestion
      - FOREST_F3_PERMANENT_PARTICIPATING_MINER_ADDRESSES=${MINER_ACTOR_ADDRESS}
```